docs_input/api/manipulation/rearranging/overlap.rst

```diff
@@ -0,0 +1,33 @@
+.. _overlap_func:
+
+overlap
+#######
+
+Create an overlapping view an of input operator giving a higher-rank view of the input
+
+For example, the following 1D tensor [1 2 3 4 5] could be cloned into a 2d tensor with a
+window size of 2 and overlap of 1, resulting in:
+ [1 2
```

Comment:
Doesn't render the rows correctly.

Reply:
This is an rst file, so it should just be newlines as it's shown there